src/app/copy/index.ts

```diff
@@ -0,0 +1,5 @@
+export { CopyModule } from './copy.module';
+export { CopyService } from './copy-service/copy.service';
+export { BlockCopyComponent } from './block-copy/block-copy.component';
+export { InlineCopyComponent } from './inline-copy/inline-copy.component';
+
```

Comment:
I'd like each component to have its own index.js file. This file would then look more like this:

```
export { 
  CopyService,
  CopyServiceModule
} from './copy-service/index';

export { 
  BlockCopyComponent,
  BlockCopyModule
} from './block-copy/index';

export {
  InlineCopyComponent,
  InlineCopyModule
} from './inline-copy/index';
```